src/main/java/org/powerflows/dmn/engine/evaluator/expression/provider/juel/PowerflowsELContext.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.powerflows.dmn.engine.evaluator.expression.provider.juel;
+
+import javax.el.ArrayELResolver;
+import javax.el.BeanELResolver;
+import javax.el.CompositeELResolver;
+import javax.el.ELContext;
+import javax.el.ELResolver;
+import javax.el.FunctionMapper;
+import javax.el.ListELResolver;
+import javax.el.MapELResolver;
+import javax.el.ResourceBundleELResolver;
+import javax.el.ValueExpression;
+import javax.el.VariableMapper;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PowerflowsELContext extends ELContext {
```

Comment:
I suggest to avoid using the project name as a part of class name. And in this case since the class is in juel package maybe better name would be JuelELContext

Reply:
ELContext implies JUEL, I'll think about this